### ISSUE TITLE
can "source" parameter allow filenames?

### ISSUE THREAD

**Issue by chharvey** [kss-node/kss-node#480]
Using gulp (related: #161) with an options json file. Can `options.source` allow filenames in addition to directories?

gulpfile.js:
```js
const gulp = require('gulp')
const kss = require('kss')

gulp.task('styleguide', async function () {
  return kss(require('kss-config.json'));
});
```

kss-config.json:
```json
{
  "source": "./node_modules/imported_package/css/styleguide.scss"
}
```

error:
```
Error: ENOTDIR: not a directory, scandir '███/node_modules/imported_package/css/styleguide.scss'
```

The reason I want to list an individual file is that **I don’t want all the files** in `./node_modules/imported_package/css/`, I want to be able to pick and choose the files I need.

**Comment by ryuran** [comment]
#433 could answer to this feature request.

**Comment by chharvey** [comment]
#433 was closed in favor of #495
